### PR TITLE
"Try it out" menubar item should be selected when on REPL 2

### PR DESCRIPTION
**Pull request by chasestarr**
Fixes issue here: https://github.com/babel/website/issues/1324

An interesting thing I learned about Liquid's `contains` operator is that it can match based on an array. The PR here adds a `match` field to each navigation item. This does seem to duplicate information (which is unexciting), but since the "try it out" menu item is intended to be overloaded I think it's a reasonable solution.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
       <ul class="nav navbar-nav">
         {% for item in site.data.navigation %}
           {% if item.subnav %}
-            <li class="dropdown {% if page.url contains item.url %}active{% endif %}">
+            <li class="dropdown {% if page.url contains item.match %}active{% endif %}">
```
Comment (Daniel15) [inline _includes/header.html]: What if we make `match` optional and fall back to `url` when it's not provided. Something like this (untested, might need some tweaking):

```
{% assign match = item.match %}
{% if match == blank %}
  {% assign match = item.url %}
{% endif %}
...
<li class="dropdown {% if page.url contains match %}active{% endif %}">
```

Comment (chasestarr) [inline _includes/header.html]: Hey - good idea. Just pushed a change 👍

**Comment by Daniel15** [comment]
Checked a few pages and it *almost* seems to work fine.

These all work:
https://deploy-preview-1339--babel.netlify.com/learn-es2015/
https://deploy-preview-1339--babel.netlify.com/docs/editors
https://deploy-preview-1339--babel.netlify.com/docs/usage/babel-register/
https://deploy-preview-1339--babel.netlify.com/repl/
https://deploy-preview-1339--babel.netlify.com/blog/
https://deploy-preview-1339--babel.netlify.com/faq/
https://deploy-preview-1339--babel.netlify.com/repl2/ <-- Awesome, it is fixed
https://deploy-preview-1339--babel.netlify.com/php/

Blog posts don't, though: 
https://deploy-preview-1339--babel.netlify.com/blog/2017/08/16/contributing-to-babel-three-lessons-to-remember

"Blog" is not highlighed. Compare to the live site, where it is: 
https://babeljs.io/blog/2017/08/16/contributing-to-babel-three-lessons-to-remember

I think this is because it's no longer matching substrings. The old code `page.url contains item.url` would match if the page URL is `/blog/foo/bar` (as `/blog/` is contained within it), but the new code `match contains page.url` does not, as the order is reversed (and `/blog/foo/bar` is of course not contained within `/blog/`)

**Comment by hzoo** [comment]
This works now on the new site with a recent docusarus change http://new.babeljs.io/repl, thanks!

**Pull request closed without merging**